examples/paycrest/.env.example

```
@@ -0,0 +1,9 @@
+PAYCREST_API_KEY=your_paycrest_api_key_here
+RPC_URL=https://starknet-mainnet.public.blastapi.io
+PRIVATE_KEY=0x...
+WALLET_ADDRESS=0x...
+
+# Recipient bank account (NGN example via GTB)
+RECIPIENT_INSTITUTION=GTBINGLA
+RECIPIENT_ACCOUNT_IDENTIFIER=0123456789
+RECIPIENT_ACCOUNT_NAME=John Doe
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RECIPIENT_ACCOUNT_NAME key should go before the RECIPIENT_INSTITUTION key

(UnorderedKey)